value-fixture/test-java-11/org/immutables/fixture/nullable/NullableAttributesTest.java

```diff
@@ -330,4 +333,32 @@ public String b() {
     check(s.a()).is("a");
     check(s.b()).is("b");
   }
+
+  @Test
+  public void nullValuesInJdkMapErrorMsg() {
+    Map<String, String> properties = new HashMap<>();
+    properties.put("a", "a val");
+    properties.put("b", null);
+    properties.put("c", "c val");
+    try {
```

Comment:
@elucash would it make sense to introduce [AssertJ](https://assertj.github.io/doc/) to the project to have more fluent assertion checks? This particular example could then be simplified to 
```
assertThatThrownBy(() -> ImmutableEntityWithMap.builder().properties(properties).build())
        .isInstanceOf(NullPointerException.class)
        .hasMessage("properties value for key: b");
```
There are also a lot of other good examples around Exception checks in https://github.com/assertj/assertj-examples/blob/main/assertions-examples/src/test/java/org/assertj/examples/ExceptionAssertionsExamples.java.

Let me know what you think. If you think this could be useful, then I could provide a PR with some example test code to show AssertJ usage and how it could make writing tests more fluent/easier to check.

Reply:
We have discussed using AssertJ or Google's Truth assertions frameworks in immutables before. For now we have decided to still use `check()` API.